src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -67,18 +66,17 @@ const getInstallationsWithAdmin = async (
 	gitHubUserClient: GitHubUserClient,
 	log: Logger,
 	login: string,
-	installations: Octokit.AppsListInstallationsForAuthenticatedUserResponseInstallationsItem[] = [],
-	jiraHost: string,
-	gitHubAppId?: number
+	installations: Octokit.AppsListInstallationsForAuthenticatedUserResponseInstallationsItem[] = []
 ): Promise<InstallationWithAdmin[]> => {
 	return await Promise.all(installations.map(async (installation) => {
 		const errors: Error[] = [];
-		const gitHubClient = await createInstallationClient(installation.id, jiraHost, log, gitHubAppId);
 
-		const numberOfReposPromise = await gitHubClient.getNumberOfReposForInstallation().catch((err) => {
-			errors.push(err);
-			return 0;
-		});
+		const numberOfReposPromise = await gitHubUserClient.getNumberOfReposForInstallation(installation.id)
+			.then((response) => response.data.total_count).
```

Comment:
So we're not using getNumberOfReposForInstallation that's defined in the installation client anymore? If not, we should probably delete later on but for now, this is prod code, we should probably feature flag this. Leave the original implementation if the flag is off and only call this new endpoint if it's on.

Reply:
We can add FF to toggle this, but I feel its unnecessary, cause both endpoints give the same result. 
The previous one used JWT token through graphql request, and this one uses githubToken through a [GET request](https://docs.github.com/en/rest/apps/installations#list-repositories-accessible-to-the-user-access-token).